packages/story-editor/src/components/panels/design/pageAttachment/pageAttachment.js

```diff
@@ -106,14 +110,31 @@ function PageAttachmentPanel() {
     }
   }, [url, displayWarning]);
 
+  const debouncedUpdate = useDebouncedCallback((props) => {
+    updatePageAttachment(props);
+  }, 300);
```

Comment:
```suggestion
  const debouncedUpdate = useDebouncedCallback(updatePageAttachment, 300);
```
Saving a few bytes ;) (+declaration needs to be moved up)